assets/decks/Superflux.csv

```diff
@@ -0,0 +1,45 @@
+Deck,Directory,File,Suit,Name,Description
+SuperFlux,SuperFlux/,Academy,Trumps,The Academy,"Doctrine, Rigor, systems"
+SuperFlux,SuperFlux/,Activist,Trumps,The Activist,"Revelatory, Optimism, Persuasion"
+SuperFlux,SuperFlux/,Assessment,Trumps,Assessment,"Reckoning, Investigation, Analysis"
+SuperFlux,SuperFlux/,Authority,Trumps,Authority,"Alpha & Omega, Domination, Absoluteness"
+SuperFlux,SuperFlux/,Capital,Trumps,Capital,"Current, Guile, Gumption"
+SuperFlux,SuperFlux/,Collapse,Trumps,Collapse,"Cataclysm, Short- sightedness, Babel"
+SuperFlux,SuperFlux/,Consumer,Trumps,The Consumer,"Gullible, Oblivious, Impulsive"
+SuperFlux,SuperFlux/,Contract,Trumps,The Contract,"Reckless faith, Security, Bondage"
+SuperFlux,SuperFlux/,Designer,Trumps,The Designer,"Alchemy, Empower, Solution"
+SuperFlux,SuperFlux/,Disruption,Trumps,Disruption,"Reckless faith, Constriction, Bondage"
+SuperFlux,SuperFlux/,Drone,Trumps,The Drone,"Strategic sight, Scrutiny, Perspective"
+SuperFlux,SuperFlux/,Growth,Trumps,Growth,"Resource, Progress, Abundance"
+SuperFlux,SuperFlux/,Hacker,Trumps,The Hacker,"Transformative mastery, Intelligence wedded to will, Cunning"
+SuperFlux,SuperFlux/,Innovation,Trumps,Innovation,"Morphogenesis, Transformation, Severance"
+SuperFlux,SuperFlux/,Market,Trumps,The Market,"Opportunity, Contingency, Predictability"
+SuperFlux,SuperFlux/,Meme,Trumps,The Meme,"Contrarian, Double-sided mirror, Hallucinatory"
+SuperFlux,SuperFlux/,Researcher,Trumps,The Researcher,"Cerebral luminosity, Transparent, Synthesizing"
+SuperFlux,SuperFlux/,RogueState,Trumps,The Rogue State,"Sacrifice, Reparation, Redemption"
+SuperFlux,SuperFlux/,State,Trumps,The State,"Hierarchy, Constraints, Stability"
+SuperFlux,SuperFlux/,Treaty,Trumps,The Treaty,"Transcendence, Perseverance, Patience"
+SuperFlux,SuperFlux/,Visionary,Trumps,The Visionary,"Illumination, Penetrate, Far-sighted"
+SuperFlux,SuperFlux/,Whistleblower,Trumps,The Whistleblower,"Circumspection, Prudence, Marginality"
+SuperFlux,SuperFlux/,R_Academy,Trumps,The Inverse Academy,"Disobedience, Neglect, Dogma"
```

Comment:
See this file for the more compact way of adding a "reversed" meaning.

https://github.com/yrevocnu/psychedelic-glass-bead-game/blob/master/assets/decks/yrevocnu.csv

Is this adequate for the task of the reversed SupeFlux cards? If so, I'd stick with it.

Reply:
I resolved the file, how does it look?